### PR TITLE
PPD Exp: updates ruby version and alpine version

### PR DESCRIPTION
**Pull request by DanielCouzens**
Updates PPD app to 2.8.0
Updates ruby to 2.7.8
Updates alpine 3.16

### CODE REVIEW

File: app/lib/version.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # frozen_string_literal: true
 
 module Version
-  MAJOR = 1
-  MINOR = 7
-  PATCH = 9
+  MAJOR = 2
```
Comment (jonrandahl) [inline app/lib/version.rb]: @DanielCouzens this is not the same as the major cadence as the version in the CHANGELOG?

**Review by jonrandahl — CHANGES_REQUESTED** [review]
Minor adjustment needed for versioning!

**Review by jonrandahl — CHANGES_REQUESTED** [review]
Sorry @DanielCouzens the major version is fine at 1, it was the version.rb configuration which needed to mirror the CHANGELOG, not the other way around! We're only updating the minor cadence as we've not jumped to the major update of v3 for Ruby yet!

File: app/lib/version.rb (hunk 1 above)
Comment (DanielCouzens) [reply app/lib/version.rb]: @jonrandahl what should the cadence be

It is 1.7.9 in dev

What should I change it to?

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,10 @@ This app allows the user to explore HMLR price-paid open linked data.
 
 ## Changelog
 
+## 2.8.0 - 2024-10
```
Comment (jonrandahl) [inline CHANGELOG.md]: @DanielCouzens as requested this should be v1.8.0

File: app/lib/version.rb (hunk 1 above)
Comment (jonrandahl) [reply app/lib/version.rb]: @DanielCouzens v1.8.0

**Review by jonrandahl — APPROVED** [review]
LGTM! 👍

**Pull request merged**